dev/percy/index.js

```diff
@@ -1,11 +1,10 @@
-import panelTest from './panelTest.json';
-import histogram from './histogram.json';
-import histogram2d from './histogram2d.json';
-import pie from './pie.json';
-import violin from './violin.json';
-import bar from './bar.json';
-import box from './box.json';
-import waterfall from './waterfall.json';
-import sunburst from './sunburst.json';
-
-export {panelTest, histogram, histogram2d, pie, violin, bar, box, waterfall, sunburst};
+export {default as panelTest} from './panelTest.json';
+export {default as histogram} from './histogram.json';
+export {default as histogram2d} from './histogram2d.json';
+export {default as pie} from './pie.json';
+export {default as violin} from './violin.json';
+export {default as bar} from './bar.json';
+export {default as box} from './box.json';
+export {default as waterfall} from './waterfall.json';
+export {default as sunburst} from './sunburst.json';
+export {default as sankey} from './sankey.json';
```

Comment:
I think this syntax is easier, since you don't need to add something in two places. What do you guys think? Shall we put it in more places?
And no, Vera, you can't just do `export sankey from...`

Reply:
yeah, I know :) that's because the sankey.json file doesn't have an export default = ...

Reply:
No, actually a default export is all there is, just syntax doesn't seem to allow and 'export from' that way...

Reply:
Can you export * from ... ?

Reply:
Well, in case of JSON they won't be named...